RecoMuon/TrackingTools/src/MuonServiceProxy.cc

```diff
@@ -38,8 +41,16 @@ MuonServiceProxy::MuonServiceProxy(const edm::ParameterSet& par):theTrackingGeom
   vector<string> propagatorNames;
 
   theMuonNavigationFlag = par.getUntrackedParameter<bool>("UseMuonNavigation",true);
-  if(theMuonNavigationFlag) theRPCLayer = par.getParameter<bool>("RPCLayers");
-  else theRPCLayer = true;
+  if(theMuonNavigationFlag) {
+                            theRPCLayer = par.getParameter<bool>("RPCLayers");
+                            theCSCLayer = par.getUntrackedParameter<bool>("CSCLayers",true);
```

Comment:
these will change behavior of the MuonNavigationSchool constructed below and should be made tracked